inventory.lua

```diff
@@ -19,6 +45,27 @@ local function send_message(pos, action, stack, from_slot, to_slot, side)
 		-- Duplicate the vector in case the caller expects it not to change.
 		side = side and vector.new(side)
 	}
+
+	-- Check if we need to include the current signal into batch
+	-- Store "prev_time" in metadata as a string to avoid integer overflow
+	local prev_time = tonumber(meta:get_string("prev_time")) or 0
+	local cur_time = minetest.get_us_time()
+	meta:set_string("prev_time", tostring(cur_time))
```

Comment:
Why do we need to store the time? It's such short intervals that temporary tracking in a variable should be sufficient.

Reply:
Done (moved to local variable)